packages/venia-concept/src/reducers/cart.js

```diff
@@ -51,6 +53,31 @@ const reducerMap = {
             }))
         };
     },
+    [actions.addItem.request]: state => {
+        return {
+            ...state,
+            addToCartActiveRequest: true
+        };
+    },
+    [actions.addItem.receive]: (state, { error }) => {
```

Comment:
Did this reducer not handle `addItem.request` and `addItem.receive` previously?

Reply:
@supernova-at do you remember what you meant by this comment? I couldn't figure it out >.<

Reply:
Oh I just noticed that these were added in this changeset and I was a bit surprised that this reducer wasn't already doing something when these things happened.

No changes needed.